bash_completion

```diff
@@ -1694,7 +1695,7 @@ _comp_compgen_ip_addresses()
     local PATH=$PATH:/sbin
     local addrs
     _comp_compgen -v addrs split -- "$({
-        LC_ALL=C ifconfig -a || ip -c=never addr show || ip addr show
+        ip -c=never addr show || ip addr show || LC_ALL=C ifconfig -a
```

Comment:
Similarly as above -- `_comp_compgen_ip_addresses` is not affected by interface name truncation I think, so let's not make this change or add a comment in the commit message why we're doing it.

Reply:
I added reasoning in the commit message.